docs/sources/tempo/configuration/grafana-agent/automatic-logging.md

```diff
@@ -19,15 +21,15 @@ Automatic logging provides an easy and fast way of discovering trace IDs
 through log messages. Well-formatted log lines are written to a Loki instance
 or to `stdout` for each span, root, or process that passes through the tracing
 pipeline. This allows for automatically building a mechanism for trace
-discovery. On top of that, we also get metrics from traces using Loki, and
+discovery. On top of that, you can also get metrics from traces using Loki, and
 allow quickly jumping from a log message to the trace view in Grafana.
 
-While this approach is useful, it isn't as powerful as [TraceQL]({{< relref
-"../../traceql" >}}). If you are here because you know you want to log the
+While this approach is useful, it isn't as powerful as TraceQL.
+If you are here because you know you want to log the
 trace ID, to enable jumping from logs to traces, then read on!
 
 If you want to query the system directly, read the [TraceQL
-documentation]({{< relref "../../traceql" >}}). 
+documentation](https://grafana.com/docs/tempo/<TEMPO_VERSION>/traceql).
```

Comment:
How does `<TEMPO_VERSION>` get populated?

Reply:
It's set in the front matter of tempo/_index.md. If this page is backported, then any links go to the correct version of the Tempo documentation. For example, links within the 2.4 doc set stay within the 2.4 docs instead of moving to /latest..